README.md

```diff
@@ -14,19 +14,119 @@ The established standard Nix formatting differs considerably from the original o
 
 For more details, see the [RFC implementation tracking issue](https://github.com/NixOS/nixfmt/issues/153).
 
-## Installation
+## Installation And Usage Instructions
 
-- `nixfmt` is in nixpkgs master as of 2019-09-04: 
+### nixpkgs/NixOS
 
-      nix-env -iA nixpkgs.nixfmt
-
-- To get the most recent version, install from master:
-
-      nix-env -f https://github.com/NixOS/nixfmt/archive/master.tar.gz -i
-
-- Nix with flakes
-
-      nix profile install github:NixOS/nixfmt
+`nixfmt` was used as the basis for the official Nix formatter with a standardized formatting.
+The new formatting differs considerably from the original one.
+This is why two packages are in nixpkgs: `nixfmt-classic` with the non-standardized formatting and `nixfmt-rfc-style`.
+
+So installing this `nixfmt` is as simple as adding to the system packages:
+
+```nix
+{ pkgs, ... }:
+
+{
+  environment.systemPackages = [ pkgs.nixfmt-rfc-style ];
+}
+```
+
+### From the repository
+
+It's also possible to install `nixfmt` directly from the repository using `nix-env`.
+This allows the newest changes to be used, which may not be equal to the `nixfmt` version used to format nixpkgs, so this should be done with care!
```

Comment:
This applies to all methods here. I think there should be an additional section referring to how the `nix-shell` in Nixpkgs provides the correct version for Nixpkgs. Alternatively, let's just drop this sentence, otherwise it implies the above would be the correct version. As an upstream project we shouldn't really have to document downstream uses.

```suggestion
```